### PR TITLE
Post to workspace

### PR DESCRIPTION
**Pull request by Rikadewi**

### CODE REVIEW

File: model/Std.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package model
```
Comment (julisman) [inline model/Std.go]: rename the file name `std.go`

File: model/Workplace.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package model
```
Comment (julisman) [inline model/Workplace.go]: rename the file name `workplace.go`

File: util/notify.go [hunk 3]
```diff
@@ -1,73 +1,79 @@
...
+			if isExist(config.GlobalConfig.IncludedReasons, event.Reason) {
+				message := parseEventToMessage(event)
+				postEvent(message)
+			}
+		}
+	}
+}
+
+func parseEventToMessage(event *corev1.Event) string {
+	return fmt.Sprintf("[%s: %s] %s/%s %s", event.Type, event.Reason, event.Namespace, event.Name, event.Message)
```
Comment (julisman) [inline util/notify.go:26]: btw could we add `last seen`? WDYT?

File: model/Std.go (hunk 1 above)
Comment (Rikadewi) [reply model/Std.go]: Done!

File: model/Workplace.go (hunk 2 above)
Comment (Rikadewi) [reply model/Workplace.go]: Done

File: util/notify.go (hunk 3 above)
Comment (Rikadewi) [reply util/notify.go:26]: There are many kind of timestamp in event, such as
```

        // The time at which the event was first recorded. (Time of server receipt is in TypeMeta.)
	// +optional
	FirstTimestamp metav1.Time `json:"firstTimestamp,omitempty" protobuf:"bytes,6,opt,name=firstTimestamp"`

	// The time at which the most recent occurrence of this event was recorded.
	// +optional
	LastTimestamp metav1.Time `json:"lastTimestamp,omitempty" protobuf:"bytes,7,opt,name=lastTimestamp"`
 
	// Time when this Event was first observed.
	// +optional
	EventTime metav1.MicroTime `json:"eventTime,omitempty" protobuf:"bytes,10,opt,name=eventTime"`
```

Comment (Rikadewi) [reply util/notify.go:26]: And when I try to print these values, I got something like this

`2020/06/12 11:27:06 [Warning: BackOff] business-acceleration/mitra-pos-backend-staging-queue-bulk-insert-product-57d6c628qvf.16109a9758acbf82 Back-off restarting failed container
FirstTimestamp: 2020-05-20 09:10:43 +0700 WIB | LastTimestamp: 2020-06-12 11:24:58 +0700 WIB | EventTime: 0001-01-01 00:00:00 +0000 UTC
`

I'm not sure what FirstTimestamp means, but I think LastTimestamp maybe usefull. Or should I include FirstTimestamp to the message too?

Comment (julisman) [reply util/notify.go:26]: `lastTImestmap` will be good

Comment (julisman) [reply util/notify.go:26]: can you print `EventTime`? I want to see the different between them

Comment (julisman) [reply util/notify.go:26]: let's have different `pr` for this feature

**Pull request merged**